.github/workflows/push.yml

```diff
@@ -41,3 +41,10 @@ jobs:
         run: |
           docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+      - name: Copy otp jar artifact to S3 bucket
+        env:
+          JAR_FILE: otp_${{ github.sha }}.jar
+          BUCKET_NAME: s3://${{ S3_BUCKET_NAME }}
+        run: |
+          cp ./target/otp-1.5.0-SNAPSHOT-shaded.jar $JAR_FILE
```

Comment:
to próba zmienienia nazwy pliku? nie da się tego zrobić poprzez
```
aws s3 cp ./target/otp-1.5.0-SNAPSHOT-shaded.jar $BUCKET_NAME/$JAR_FILE
```

Reply:
I just checked it in documentation, and it is possible to do it like suggested. Fixed.